Kinvey-Xamarin/Store/ReadRequest.cs

```diff
@@ -60,16 +60,11 @@ protected string BuildMongoQuery()
 			{
 				StringQueryBuilder queryBuilder = new StringQueryBuilder();
 
-				queryBuilder.Reset();
```

Comment:
Because the constructor for `StringQueryBuilder` itself calls `Reset()`. So in effect, it was being called twice. Harmless either way, but unnecessary.